Doc/whatsnew/3.7.rst

```diff
@@ -604,3 +604,21 @@ CPython bytecode changes
 
 * Added two new opcodes: :opcode:`LOAD_METHOD` and :opcode:`CALL_METHOD`.
   (Contributed by Yury Selivanov and INADA Naoki in :issue:`26110`.)
+
+
+Documentation
+=============
+
+.. _whatsnew37-pep545:
+
+PEP 545: Python Documentation Translations
+------------------------------------------
+
+There's now a PEP describing how the Python documentation is
```

Comment:
Maybe replace "a PEP" with :pep:`545` to get a link.

Reply:
that's right. fixed, and had to reword for consistency.